app/views/api/v0/articles/me.json.jbuilder

```diff
@@ -0,0 +1,46 @@
+json.array! @articles do |article|
+  json.type_of            "article"
+  json.id                 article.id
+  json.title              article.title
+  json.description        article.description
+  json.cover_image        cloud_cover_url(article.main_image)
+  json.published          article.published
+  json.published_at       article.published_at
+  json.tag_list           article.cached_tag_list_array
+  json.slug               article.slug
+  json.path               article.path
+  json.url                article.url
+  json.canonical_url      article.processed_canonical_url
+  json.comments_count     article.comments_count
+  json.positive_reactions_count article.positive_reactions_count
+  json.published_timestamp article.published_timestamp
+  json.markdown            article.body_markdown
```

Comment:
can we keep this called `body_markdown` ? It's how we call it everywhere, in the [show.json.jbuilder](https://github.com/thepracticaldev/dev.to/blob/master/app/views/api/v0/articles/show.json.jbuilder#L24)

Reply:
Yep I'll do that.